### PR TITLE
in non-production environments, use basic-vpc as the s3 plan

### PR DESCRIPTION
**Pull request by drewbo**
## Changes proposed in this pull request:
- in non-production environments, use basic-vpc as the s3 plan

## security considerations
Restricts S3 access (only in staging for now) to requests that originate from within the cloud.gov VPC

### CODE REVIEW

**Review by apburnes — DISMISSED** [review]

File: api/services/SiteCreator.js [hunk 1]
```diff
@@ -111,7 +113,9 @@ function buildInfrastructure(params, s3ServiceName) {
   return apiClient.createSiteBucket(
     s3ServiceName,
     config.env.cfSpaceGuid,
-    config.app.s3ServicePlanId
+    config.app.s3ServicePlanId,
+    null,
```
Comment (apburnes) [inline api/services/SiteCreator.js:117]: I think function arguments need to be `undefined` and not null in order to have the default argument value be used . Would it make more sense putting the feature flag in the `createSiteBucket` function?

Comment (drewbo) [reply api/services/SiteCreator.js:117]: Good call on `undefined`. I can move the check to the `createSiteBucket` function also

File: test/api/unit/utils/cfApiClient.create.test.js [hunk 2]
```diff
@@ -172,6 +172,7 @@ describe('CloudFoundryAPIClient', () => {
         name, config.env.cfSpaceGuid, config.app.s3ServicePlanId, keyIdentifier, planName
       )
         .then((res) => {
+          console.log(res)
```
Comment (apburnes) [inline test/api/unit/utils/cfApiClient.create.test.js:175]:
```suggestion
```

**Review by apburnes — COMMENTED** [review]
Put a suggesstion in for a rouge `console.log` in the tests. Should be good to go after that.

**Review by apburnes — APPROVED** [review]
🪰

**Pull request merged**